A/ADIOS2/build_tarballs.jl

```diff
@@ -146,7 +146,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency(PackageSpec(name="Blosc2_jll")),
+    Dependency(PackageSpec(name="Blosc2_jll"); compat="201.1500.0"),
```

Comment:
Need to also change the version number of the package

Reply:
d'oh.